app/models/ingest_job.rb

```diff
@@ -838,8 +838,14 @@ def launch_anndata_subparse_jobs
         job.delay.push_remote_and_launch_ingest
       end
 
-      # unset anndata_summary flag to allow reporting summary later
-      study_file.unset_anndata_summary!
+      # if this was only a raw counts extraction, update parse status
+      if params_object.extract == %w[raw_counts]
+        study_file.update(parse_status: 'parsed')
```

Comment:
If a user uploads a big AnnData file, then _while that large ingest is ongoing_ indicates in the UI that the file has raw counts, will this line falsely suggest their original larger file is parsed when it hasn't yet finished?

That seems like a plausible scenario.  I'm less confident about severity, but I suspect it'd at least be confusing while the larger ingest is ongoing.  In any case, impact seems low at worst, and think the current change is net benefit.

So I don't consider this a blocker, just worth keeping in mind.

Reply:
That's a decent point... the save buttons will be disabled for the most part, but the one in the AnnData tab will still work.  In theory they could use that to save.  The problem is that it won't actually do the raw counts extraction because it will first check to see if it's already parsed, and since this will be false it will exit silently while still saving the update.  I coded it this way because I didn't want the opportunity of concurrent extractions potentially failing and causing downstream issues.  If we don't check the `parse_status` flag, then any update while the extraction is going would spawn a new ingest process, and then they'd all fail.  I'm loathe to add more status flags - we're already replete with them, and they're not terribly reliable, especially in an asynchronous background process. 
 Given that the save button in the expression form is disabled until the main extractions are done, I don't think this is a large issue, but we can keep an eye on it to see if it becomes a problem.